rosdep/base.yaml

```diff
@@ -125,6 +125,12 @@ aravis-dev:
     '*': [libaravis-dev]
     bionic: null
     xenial: null
+aravis-tools:
+  debian: [aravis-tools]
+  ubuntu:
+    '*': [aravis-tools]
+    bionic: null
+    xenial: null
```

Comment:
You can drop the `xenial` key now; Xenial is End-of-Life.

Reply:
@clalancette  amended